test/e2e/functional/tests/e2e.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/metallb/metallb-operator/test/consts"
 	testclient "github.com/metallb/metallb-operator/test/e2e/client"
 	metallbutils "github.com/metallb/metallb-operator/test/e2e/metallb"
-	admv1 "k8s.io/api/admissionregistration/v1"
+	admv1 "k8s.io/api/admissionregistration/v1beta1"
```

Comment:
is this necessary?

Reply:
that has been fixed there was prob with controller gen tool version we had we have to pump to 0.7.0 to sort this version issue out